common/src/main/resources/assets/exampleplugin/lang/en_us.json

```
@@ -2,5 +2,9 @@
   "goofyfiguraplugin.gui.goofyscreen.ok.tooltip": "Press this to return to the previous screen",
   "goofyfiguraplugin.gui.goofyscreen": "Example Screen",
   "goofyfiguraplugin.permissions.value.example_permission": "Example Permission",
-  "goofyfiguraplugin.permissions.value.example_permission.tooltip": "This is an example for a permission"
+  "goofyfiguraplugin.permissions.value.example_permission.tooltip": "This is an example for a permission",
+  "figura.docs.goofy": "The dumping ground of the figura goofballs",
+  "figura.docs.globals.goofy": "The global instance of GoofyAPI",
+  "figura.docs.goofy.regex_match": "Returns a list of matches based on the specified\npattern.",
+  "figura.docs.goofy.regex_sub": "Returns a string with all matches of the specified\npattern replaced."
 }
```

no ending EOL; some programs will complain
```suggestion
}
```